cmd/tk/env.go

```diff
@@ -196,6 +197,18 @@ func envRemoveCmd() *cli.Command {
 	}
 }
 
+// simple wrapper for labels.Labels to use our map in their querier
+type labelset map[string]string
+
+func (l labelset) Has(label string) (exists bool) {
+	_, exists = l[label]
+	return exists
+}
+
+func (l labelset) Get(label string) (value string) {
+	return l[label]
+}
+
```

Comment:
This is not much code and `spec.Metadata.Labels` is nothing else but a labelset, so I that's a good place to move these to.